pages/governance.md

```diff
@@ -0,0 +1,187 @@
+discussions about JuMP.
+
+The current chatroom is the [JuMP-dev Gitter channel](https://gitter.im/JuliaOpt/JuMP-dev).
+
+### Monthly developer call
+
+The Steering Council will maintain a monthly developer call to discuss JuMP-related
+business.
+
+The calls are currently scheduled on the fourth Thursday of every month at 14:00
+Eastern. For information on how to take part, join the [Developer chatroom] and
+ask `@odow` or `@mlubin` for an invite to the monthly developer call.
+
+### JuMP-dev workshops
+
+The Steering Council will periodically choose a Chair to organize a JuMP-dev
+workshop.
+
+Previous workshops include [Santiago, 2019](https://jump.dev/meetings/santiago2019/),
+[Bordeaux, 2018](https://jump.dev/meetings/bordeaux2018/), and [Boston, 2017](https://jump.dev/meetings/mit2017/).
+
+Announcements of future workshops will be communicated through the
+[Community forum] and via the [jump.dev](https://jump.dev) website.
+
+## Commit rights
+
+Commit rights are managed through [Github permissions](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization)
+in the [JuMP-dev Github organization].
+
+There are two levels of commit rights: repository maintainers and core
+contributors.
+
+For security, all maintainers and core contributors must have two-factor
+authentication enabled through their Github account.
+
+### Repository Maintainer
+
+Repository maintainers have `Maintain` rights to an individual repository as an
+outside collaborator. Among other rights, this allows them to push code to a
+branch in the `jump-dev` organization instead of their personal fork, merge pull
+requests, and close issues.
+
+Community members can become a repository maintainer by actively engaging in the
+development and maintenance of a repository for a sustained period of time. This
+includes triaging issues, proposing and reviewing pull requests, and updating
+any binary dependencies as needed.
+
+If they judge it is in the best interest of the repository, any Core Contributor
+has individual authority to give a community member `Maintain` rights with
+notification to the other core contributors.
+
+Community members may be maintainers of multiple repositories at the same time.
+
+### Core Contributor
+
+Core contributors are added as members of the [JuMP-dev Github organization] and
+have `Admin` permission to every repository.
+
+A new core contributor may be added by consensus of the current core
+contributors and notification to the Steering Council. Before becoming a core
+contributor, it is expected that the community member has already recieved
+`Maintain` rights to several repositories.
+
+The current core contributors are:
+
+* [Miles Lubin](https://github.com/mlubin)
+* [Benoît Legat](https://github.com/blegat)
+* [Joaquim Dias Garcia](https://github.com/joaquimg)
+* [Joey Huchette](https://github.com/joehuchette)
```

Comment:
Is there a reason @joehuchette has `Member` permission in JuMP-dev and not `Owner`?

Reply:
A related question: @joehuchette, would you prefer to be listed under core contributors or emeritus core contributors?

Reply:
A bit conflicted, mostly because "emeritus" suggests retiring from contributing in perpetuity, which is not my intention. I'll defer to the BDFL :)

Reply:
The only mechanism defined for transitioning from being a core developer to an emeritus core developer is by request, so you'll remain a core developer until you ask not to be :)